README.md

```diff
@@ -37,6 +37,12 @@ _Default:_ `false`
 
 Specify if completed directory names should include a trailing slash. Enabling this option makes this source behave like Vim's built-in path completion.
 
+### visual_slash (type: boolean)
```

Comment:
label_trailing_slash is better name.
and default value seems to,wrong.

Reply:
Sure, change it to `label_trailing_slash` if you wish. I changed the default value to true which shows it, but in my config I will set it to false. Set the default to false if you think that's a better default.

Reply:
Yes. please change :)

Reply:
Also, should the default be `true` or `false`?

Reply:
label_trailing_slash should be true by defailt

Reply:
Okay, the latest commit should be good.